jenkins-dsl/cassandra_job_dsl_seed.groovy

```diff
@@ -84,10 +79,48 @@ def testSplits = 8
 def dtestSplits = 64
 def dtestLargeSplits = 8
 
+def exists(branchName, targetName) {
+    switch (targetName) {
+        case 'artifact':
+            // migrated to in-pipeline stages
+            return true; // TODO CASSANDRA-18133 // return !(branchName = 'trunk' || branchName ==~ /cassandra-5.\d+/)
```

Comment:
will be updated post [CASSANDRA-18133](https://issues.apache.org/jira/browse/CASSANDRA-18133)